### ISSUE TITLE
Ques about LearningRateDecayOptimizer

### ISSUE THREAD

**Issue by huixiancheng** [facebookresearch/ConvNeXt#97]
Hi. I am very confused about this optimizer, does it set different lr and weight_decay for different layers of Block？
https://github.com/facebookresearch/ConvNeXt/blob/d1fa8f6fef0a165b27399986cc2bdacc92777e40/semantic_segmentation/configs/convnext/upernet_convnext_base_512_160k_ade20k_ss.py#L36-L40
Also, is it possible to implement this part without relying on mmcv?
https://github.com/facebookresearch/ConvNeXt/blob/d1fa8f6fef0a165b27399986cc2bdacc92777e40/semantic_segmentation/mmcv_custom/layer_decay_optimizer_constructor.py#L58-L71

**Comment by liuzhuang13** [comment]
Hi,

The learning rate is stage_wise for fine-tuning (each stage has a different lr). But weight decay is not. There is only one weight decay value, but for certain parameters we do not apply weight decay like baises.

Yes it is possible, for example, in our ImageNet classification code part we did not rely on mmcv but also uses a layer-wise lr decay.

**Comment by huixiancheng** [comment]
thx for your explain.
